<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountSettingsWidget</class>
 <widget class="QWidget" name="AccountSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* --- Main Window Style --- */
QWidget#AccountSettingsWidget {
    background-color: #2f3136;
    color: #e8e9ea;
    font-family: &quot;Segoe UI&quot;, Roboto, Cantarell, &quot;Helvetica Neue&quot;, sans-serif;
    font-size: 14px;
}

/* --- Content Card Style --- */
QWidget#contentCard {
    background-color: #36393f;
    border-radius: 10px;
    border: 1px solid #202225;
}

/* --- Layouts --- */
QVBoxLayout, QHBoxLayout, QFormLayout, QGridLayout {
    background-color: transparent;
}

/* --- Section Card Style --- */
QFrame.sectionCard {
    background-color: #40444b;
    border-radius: 8px;
    border: 1px solid #202225;
    padding: 15px;
}

/* --- Avatar Style --- */
QLabel#avatarLabel {
    background-color: #5865F2;
    border: 3px solid #7289da;
    border-radius: 60px;
    min-width: 120px;
    max-width: 120px;
    min-height: 120px;
    max-height: 120px;
    qproperty-alignment: AlignCenter;
    color: white;
    font-size: 42px;
    font-weight: bold;
}

/* --- Heading Styles --- */
QLabel#usernameLabel {
    color: white;
    font-size: 22px;
    font-weight: bold;
    background-color: transparent;
}

QLabel#emailLabel {
    color: #b9bbbe;
    font-size: 16px;
    background-color: transparent;
}

QLabel#positionLabel {
    color: #72767d;
    font-size: 14px;
    font-style: italic;
    background-color: transparent;
}

QLabel.sectionTitle {
    color: white;
    font-size: 16px;
    font-weight: bold;
    padding-bottom: 5px;
    border-bottom: 1px solid #4f545c;
    background-color: transparent;
}

/* --- Form Labels --- */
QLabel.formLabel {
    color: #b9bbbe;
    font-size: 14px;
    font-weight: 500;
    padding-right: 5px;
    background-color: transparent;
}

/* --- Line Edits --- */
QLineEdit {
    background-color: #40444b;
    color: #dcddde;
    border: 1px solid #202225;
    border-radius: 4px;
    padding: 8px 12px;
    font-size: 14px;
    selection-background-color: #5865F2;
}

QLineEdit:focus {
    border: 1px solid #5865F2;
    background-color: #454950;
}

QLineEdit:disabled {
    background-color: #36393f;
    color: #72767d;
}

QLineEdit::placeholder {
    color: #72767d;
}

/* --- Buttons --- */
QPushButton.primaryButton {
    background-color: #5865F2;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: 600;
}

QPushButton.primaryButton:hover {
    background-color: #4752c4;
}

QPushButton.primaryButton:pressed {
    background-color: #3c45a5;
}

QPushButton.secondaryButton {
    background-color: #4f545c;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 16px;
    font-size: 13px;
}

QPushButton.secondaryButton:hover {
    background-color: #5d6269;
}

QPushButton.secondaryButton:pressed {
    background-color: #6b6f79;
}

/* --- Status Label --- */
QLabel#statusLabel {
    color: #5865F2;
    font-size: 14px;
    min-height: 20px;
    background-color: transparent;
}

QLabel#statusLabel[warning=&quot;true&quot;] {
    color: #faa61a;
}

QLabel#statusLabel[error=&quot;true&quot;] {
    color: #ed4245;
}

/* --- Tooltip --- */
QToolTip {
    background-color: #202225;
    color: #dcddde;
    border: 1px solid #4f545c;
    padding: 5px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="contentCard" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="headerLayout">
        <property name="spacing">
         <number>25</number>
        </property>
        <item>
         <widget class="QLabel" name="avatarLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="profileInfo">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="usernameLabel">
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>user@example.com</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="positionLabel">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="profileSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="headerSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="emailSection">
        <property name="styleSheet">
         <string>QFrame#emailSection{
background-color: #40444b;
border-radius: 8px;
border: 1px solid #202225;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="emailSectionLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="emailSectionTitle">
           <property name="styleSheet">
            <string>color: white;
font-size: 16px;
font-weight: bold;
padding-bottom: 5px;
border-bottom: 1px solid #4f545c;</string>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="emailFormLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="currentEmailLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Aktualny Email</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="email">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="newEmailLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Nowy Email</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="new_email">
             <property name="placeholderText">
              <string>Wprowad≈∫ nowy email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="passwordSection">
        <property name="styleSheet">
         <string>QFrame#passwordSection{
background-color: #40444b;
border-radius: 8px;
border: 1px solid #202225;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="passwordSectionLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="passwordSectionTitle">
           <property name="styleSheet">
            <string>color: white;
font-size: 16px;
font-weight: bold;
padding-bottom: 5px;
border-bottom: 1px solid #4f545c;</string>
           </property>
           <property name="text">
            <string>Zmiana has≈Ça</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="passwordFormLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="currentPasswordLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Aktualne Has≈Ço</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="password">
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Wprowad≈∫ aktualne has≈Ço</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="newPasswordLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Nowe Has≈Ço</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="new_password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Wprowad≈∫ nowe has≈Ço</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="confirmPasswordLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Potwierd≈∫ Has≈Ço</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="confirm_new_password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Powt√≥rz has≈Ço</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="personalInfoSection">
        <property name="styleSheet">
         <string>QFrame#personalInfoSection{
background-color: #40444b;
border-radius: 8px;
border: 1px solid #202225;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="personalInfoSectionLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="personalInfoSectionTitle">
           <property name="styleSheet">
            <string>color: white;
font-size: 16px;
font-weight: bold;
padding-bottom: 5px;
border-bottom: 1px solid #4f545c;</string>
           </property>
           <property name="text">
            <string>Informacje osobiste</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="personalInfoFormLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Imiƒô</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="surnameLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Nazwisko</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="surname">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="jobPositionLabel">
             <property name="styleSheet">
              <string>color: #b9bbbe;
font-size: 14px;
font-weight: 500;
padding-right: 5px;</string>
             </property>
             <property name="text">
              <string>Stanowisko</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="position">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottomLayout">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="bottomSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveChanges">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>background-color: #5865F2;
color: white;
border: none;
border-radius: 4px;
padding: 10px 20px;
font-size: 14px;
font-weight: 600;</string>
          </property>
          <property name="text">
           <string>Zapisz zmiany</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>new_email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>new_password</tabstop>
  <tabstop>confirm_new_password</tabstop>
  <tabstop>saveChanges</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
